lib/datasource/repology/index.spec.ts

```diff
@@ -343,5 +344,113 @@ describe('datasource/repology/index', () => {
       expect(release).toBeNull();
       expect(httpMock.getTrace()).toMatchSnapshot();
     });
+
+    it('returns correct package types for api_call', async () => {
+      const pkgs: RepologyPackage[] = [
+        {
+          repo: 'some_repo',
+          version: '1.0.0',
+          visiblename: 'some-package',
+          srcname: 'some-package',
+        },
+        {
+          repo: 'some_repo',
+          version: '2.0.0',
+          visiblename: 'not-some-package',
+          srcname: 'not-some-package',
+        },
+        {
+          repo: 'some_repo',
+          version: '3.0.0',
+          visiblename: 'some-package',
+          srcname: 'not-some-package',
+        },
+        {
+          repo: 'some_repo',
+          version: '4.0.0',
+          visiblename: 'some-package',
+          binname: 'some-package',
+        },
+        {
+          repo: 'some_repo',
+          version: '5.0.0',
+          visiblename: 'not-some-package',
+          binname: 'not-some-package',
+        },
+        {
+          repo: 'some_repo',
+          version: '6.0.0',
+          visiblename: 'some-package',
+          binname: 'not-some-package',
+        },
+        { repo: 'some_repo', version: '7.0.0', visiblename: 'some-package' },
+        {
+          repo: 'some_repo',
+          version: '8.0.0',
+          visiblename: 'not-some-package',
+        },
+        {
+          repo: 'not_some_repo',
+          version: '9.0.0',
+          visiblename: 'some-package',
+        },
+        {
+          repo: 'not_some_repo',
+          version: '10.0.0',
+          visiblename: 'some-package',
+          srcname: 'some-package',
+        },
+        {
+          repo: 'not_some_repo',
+          version: '11.0.0',
+          visiblename: 'some-package',
+          binname: 'some-package',
+        },
+      ];
+      const pkgsJSON = JSON.stringify(pkgs);
+
+      mockResolverCall('some_repo', 'some-package', 'binname', {
+        status: 403,
+      });
+
+      mockApiCall('some-package', {
+        status: 200,
+        body: pkgsJSON,
+      });
+
+      const res = await getPkgReleases({
+        datasource,
+        versioning,
+        depName: 'some_repo/some-package',
+      });
+      expect(res).toMatchSnapshot();
```

Comment:
As res is small, make an inline `toEqual`, so it only needs one expect

Reply:
Done